schema/config-linux.json

```diff
@@ -283,8 +283,14 @@
             },
             "timeOffsets": {
                 "type": "object",
-                "additionalProperties": {
-                    "$ref": "defs-linux.json#/definitions/TimeOffsets"
+                "type": "object",
+                "properties": {
+                    "boottime": {
+                        "$ref": "defs-linux.json#/definitions/TimeOffsets"
```

Comment:
I am not a reviewer, but since I went through the trouble of checking, I will leave a comment to make it easier for other reviewers to check.
https://github.com/opencontainers/runtime-spec/commit/36bb632767af9022bd2ede974d2cadb6c318be8c#diff-d1de2574b4882bd4de1fec3238c90fd13a4ee8ce8142644ff82a5eb0d76cf69aR313-R322